### PR TITLE
Update tests to pass

### PR DESCRIPTION
**Pull request by mariakleiner**

### CODE REVIEW

File: dev/test/specs/starter-test.js [hunk 1]
```diff
@@ -216,7 +216,9 @@ function allSuggestions() {
 }
 
 function acceptSuggestion(textSubstring) {
+  wait(2);
```
Comment (smalls) [inline dev/test/specs/starter-test.js:222]: Is this required? openSuggestionDrawer() also starts with wait(2), right?

**Review by smalls — APPROVED** [review]
I wonder if we've past the point where we're making selenium more robust (good) and are now working around system bugs.

This was stable until last week, so I wonder if we've hit other issues.

That being said, it seems fine to add additional waits and we can always pull them back out later.

File: dev/test/specs/starter-test.js (hunk 1 above)
Comment (mariakleiner) [reply dev/test/specs/starter-test.js:222]: trying and wondering led to this :(
maybe i should increase the wait time in waitForStillness instead?
i could try it.

even reducing the wait here to 1 seconds causes the test (more consistently) to fail...

**Comment by mariakleiner** [comment]
travis still fails... it says "crashed"...
i commented out the --headless, maybe this is the reason?

**Comment by smalls** [comment]
I have a fix for 'crashed', I just merged it (#125). I don't think headless will work but let's see!

**Comment by mariakleiner** [comment]
I synced your fix, now i'm getting:
ERROR: unknown error: Chrome failed to start: exited abnormally

**Comment by smalls** [comment]
Hmm, that's weird. It's working in arcs.

**Comment by smalls** [comment]
Oh, maybe put headless back?

File: dev/artifacts/Products/Products.recipes [hunk 2]
```diff
@@ -7,8 +7,82 @@
 
 import 'Product.schema'
 import 'ShowProducts.manifest'
+import 'Recommend.manifest'
```
Comment (smalls) [inline dev/artifacts/Products/Products.recipes:10]: Do we need all the changes in here? This was a straight copy over from what we had in sjmiles/arc-stories.

**Review by smalls — APPROVED** [review]

File: dev/artifacts/Products/Products.recipes (hunk 2 above)
Comment (mariakleiner) [reply dev/artifacts/Products/Products.recipes:10]: we aren't using sjmiles/arc-stories for gift shopping, we were using 
https://github.com/PolymerLabs/arcs-cdn/blob/gh-pages/dev/apps/web/arc.manifest#L16
https://github.com/mmandlis/arcs-stories/blob/master/0.3/Products/Products.recipes

Comment (smalls) [reply dev/artifacts/Products/Products.recipes:10]: Ahhh. Great!

**Pull request merged**